changelogs/unreleased/add-libunwind-submodule.md

```diff
@@ -0,0 +1,6 @@
+## feature/build
+
+* Add bundling of GNU libunwind to support backtrace on aarch64-linux-gnu
+  platform.
+
```

Comment:
The new line isn't needed.

Reply:
Because of this extra new line the generated changelog (run [tools/gen-release-notes](https://github.com/tarantool/tarantool/blob/master/tools/gen-release-notes)) looks inconsistent:

```
* Added bundling of libnghttp2 for bundled libcurl to support HTTP/2
  for http client. The CMake version requirement is updated from 3.2 to 3.3.
* Support fedora-35 build. (gh-6692)
* Add bundling of GNU libunwind to support backtrace on aarch64-linux-gnu
  platform.

* Re-enable backtrace for all RHEL distributions by default (gh-4611).
```

We do need to add a check for this, either in checkpatch or in gen-release-notes.

Also, while we are at it, please note that changelog entries should be in Past Simple according to our [guideline](https://www.tarantool.io/en/doc/latest/contributing/release_notes/).

Reply:
Fixed: removed redundant line break and change changelog entry tense to 'Past Simple'.

Reply:
The extra line break is still there.

Reply:
Beg your pardon: I misunderstood you and thought you were talking about EOL at EOF.